tests/alembic_migrations/versions/v_84b7fc2596d5_add_utility_views/conftest.py

```diff
@@ -0,0 +1,31 @@
+import pytest
```

Comment:
Looking through `alembic_migrations/` there are now 3 `conftest.py` files.  Would it not be better to move them into a single file?

Reply:
That was deliberate. `conftest.py` files cascade, as you know. I am taking advantage of that here, so that the higher `conftest`s factor out common elements, and the deeper ones specialize them for the specific tests in that directory. That's what they're for, and doing this has made the test code _much_ simpler in several places.

Reply:
Ah right!  I had forgotten that they behave that way.  Makes more sense looking at it now :+1: